lib/src/mjpeg.dart

```diff
@@ -61,7 +64,15 @@ class Mjpeg extends HookWidget {
     final state = useMemoized(() => _MjpegStateNotifier());
     final visible = useListenable(state);
     final errorState = useState<List<dynamic>?>(null);
-    final manager = useMemoized(() => _StreamManager(stream, isLive && visible.visible, headers, timeout), [stream, isLive, visible.visible, timeout]);
+    final manager = useMemoized(
+        () => _StreamManager(
+              stream,
+              isLive && visible.visible,
+              headers,
+              timeout,
+              client ?? Client(),
+            ),
+        [stream, isLive, visible.visible, timeout]);
```

Comment:
Here you need to the client too, cause if the client changes, you want to recreate the manager to take that new client into account

Reply:
What do you mean exactly? I pass the client to the stream manager if it's given, Otherwise I'll create a default one.

Reply:
Do you mean, in the useMemoized cache?

Reply:
I just assumed you did, and added a commit :)

Reply:
Yes perfect like this, when user change http client for whatever reason the manager will be recreated. Thanks